### PR TITLE
Move taskmap controls to top corner

### PR DESCRIPTION
**Pull request by PatricKangasmaki**
Moved the task map controls from bottom left to top right corner. Also made the icons a bit larger, any opinions on that? Are they good now or should they be changed more (to either smaller or bigger)

### CODE REVIEW

**Comment by PatricKangasmaki** [comment]
Removed default `showFitView` from React flow controls and replaced it with MUI icon with fitView() onClick func. Added a tooltip for it as well

File: frontend/src/components/TaskMap.module.scss [hunk 1]
```diff
@@ -7,17 +7,26 @@ $ZOOM: var(--zoom, 1);
...
-  width: 16px;
+.controlsContainer {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 80px;
 }
 
-.restart {
+.controls {
+  display: flex;
```
Comment (IidaKainu) [inline frontend/src/components/TaskMap.module.scss:17]: The controls look slightly better to me centered
```suggestion
  display: flex;
  align-items: center;
```

File: frontend/src/components/TaskMap.tsx [hunk 2]
```diff
@@ -279,19 +280,38 @@ export const TaskMap: FC<{
...
-            />
-          </InfoTooltip>
-        </Controls>
+        <div className={classes(css.controlsContainer)}>
+          <Controls
+            className={classes(css.controls)}
+            showInteractive={false}
+            showZoom={false}
+            showFitView={false}
+          >
```
Comment (IidaKainu) [inline frontend/src/components/TaskMap.tsx:289]: Do we need the Controls component anymore as each control is false? 
Without this, there is no need for relative control styles (example below)

```
.controlsContainer {
  @extend .layout-row;
  justify-content: flex-end;
  align-items: center;
  margin-right: 10px;
  margin-top: 10px;
  gap: 7px;
}

.controlButton {
  cursor: pointer;
}
```

File: frontend/src/i18/english.ts [hunk 3]
```diff
@@ -419,6 +419,7 @@ export const english = {
     'Client shares score': 'Client shares score',
     Poor: 'Poor',
     Great: 'Great',
-    'Reset taskgroup positions': 'Reset taskgroup positions',
+    'Fit view -tooltip': 'Fit taskgroups to the current viewport',
+    'Taskgroup reset -tooltip': 'Reset task group positions',
```
Comment (IidaKainu) [inline frontend/src/i18/english.ts:423]: The new tooltip msg is a little long, could the following work?
```suggestion
    'Fit view -tooltip': 'Fit groups to the viewport',
    'Taskgroup reset -tooltip': 'Reset group positions',
```

**Review by IidaKainu — CHANGES_REQUESTED** [review]
The new buttons look much more noticeable, great progress 👍

File: frontend/src/components/TaskMap.tsx (hunk 2 above)
Comment (PatricKangasmaki) [reply frontend/src/components/TaskMap.tsx:289]: I also tried removing this as I noticed that I disabled all the functionalities but the buttons don't seem to work on the canvas unless they're inside the <Controls> .. 😅

File: frontend/src/i18/english.ts (hunk 3 above)
Comment (PatricKangasmaki) [reply frontend/src/i18/english.ts:423]: Yeah these seem better! I'll do the change

File: frontend/src/components/TaskMap.module.scss (hunk 1 above)
Comment (PatricKangasmaki) [reply frontend/src/components/TaskMap.module.scss:17]: This seems good. I also changed the infoIcon to be 1px larger, so I could get rid of the `marginTop: 2px` at the control buttons - now they look nicely aligned

**Comment by PatricKangasmaki** [comment]
Added your suggestions 👍

**Review by IidaKainu — APPROVED** [review]

**Pull request merged**